site/pages/docs/versions/v4.0/v4.0-changelog-en.hbs

```diff
@@ -4,11 +4,88 @@
+				<li>Patch - RDFa markup for main element - Fixed RDFa markup issue on the main element by adding <code>typeof="WebPageElement"</code> attribute to the <code>&lt;main&gt;</code> element.</li>
+			</ul>
+		</li>
+		<li>Styles and plugins
+			<ul>
+				<li>Minor - Custom CSS variant - New concept that allows integration of custom CSS for implementation variant of GCWeb by writing styles inside	new SASS files that are included in the build for pick-up.</li>
+				<li>Minor - Background colour -	Set a dark background to an element (e.g. well) with the class <code>.bg-dark</code> or a completely black one with <code>.bg-darker</code>.</li>
+				<li>Minor - Text colour- Set text in white with a CSS class called <code>text-white</code>. Works jointly with dark backgrounds.</li>
+				<li>Minor - Redacted text - Change of redacted text character to a full block when <code>.redacted</code> is in use.</li>
+				<li>Minor - Well - Provide a responsive well that takes the full width of its parent by adding the <code>.header-rwd</code> class to it.</li>
+				<li>Minor - URL manager and geomap - Added the possibility to prefill geomap configurations with action management.</li>
+				<li>Patch - Footer background image - Canadian flag was added on the top of the parliament on the landscpae image.</li>
+				<li>Patch - Carousel - Link underline override was removed on carousel.</li>
+				<li>Patch - Multimedia player - Top margin override was removed on multimedia player.</li>
+				<li>Patch - Menu - Fix menu navigation on different browsers when using a screen reader by running rigorous operability tests with JAWS and NVDA.</li>
+			</ul>
+		</li>
+		<li>Experimental
+			<ul>
+				<li>Minor - Bootstrap 4 spacers - CSS classes added for padding and margin spacing, e.g. <code>.p-0</code> for no padding and <code>.mb-sm-5</code> for extra-large bottom margin for small devices and up.</li>
+				<li>Minor - Background image size - CSS class added for a background image to cover the boxes size with <code>.bg-cover</code>.</li>
+				<li>Minor - Background image URL - New plugin that allows to set a background image from a data attribute called <code>data-bgimg</code>. It was introduced in the specifications but is not yet supported by browsers.</li>
+				<li>Minor - Theme colours - Provides the ability to change theme colours to pink or dark.</li>
+			</ul>
+		</li>
+		<li>Maintenance
+			<ul>
+				<li>Build script - HTMLlint exception was added to prohibit integrity attribute in link tags until adopted in the HTML specifications.</li>
+				<li>WET-BOEW dependency - Bumped WET-BOEW dependency in the "package.json" to 2019-05-22 build, which includes build fixes related to sprites and closure dependency for geomap, and also backward compatibility with the zoom feature on geomap.</li>
+				<li>GCWeb install from other repository - Allows to install GCWeb from "themes-cdn_GCWeb" repository.</li>
```

Comment:
Edit this line for something like: Include a package.lock in the repository "themes-cdn_GCWeb" to be able to refer at it as a dependency in a nodejs project.